daprdocs/content/en/concepts/dapr-services.md

```diff
@@ -0,0 +1,45 @@
+---
+type: docs
+title: "Dapr Services"
+linkTitle: "Services"
+weight: 100
+---
+
+
+Dapr supports multiple services that help sidecar - sidecar, sidecar - app and sidecar - system communication. Following are the services and the way to leverage them with your application.
+
+#### Daprd
+
+Dapr sidecar is available as Kubernetes container or as a Daprd process while using Dapr locally. Dapr runs Daprd by default when `Dapr Run` is executed through Dapr cli. While that's the default behavior, some users might still have to use Daprd explicity in certain special scenarios. For example a user may want to run sidecar in production system without running in a container. In cases like that when daprd can be executed through command line as follows
+
+```bash
+~/.dapr/bin/daprd -app-id <app-name>
+```
+app-id is a required argument which indicates a unique string id for your Dapr instance. Following are other arguments that can be used while executing daprd.
+
+| Options | Description |
```

Comment:
This table is duplicative of https://github.com/dapr/docs/pull/1659. We should either drop this table and point to this page, or think through how these both can co-exist

Reply:
How about we add a sentence as "For information about daprd options, please refer this daprd--help" and we can link "daprd --help". which. would convey how to know about these options as well as show the actual table?